benchmarks/tests/test_tpch_benchmark.py

```diff
@@ -13,99 +13,55 @@
   --query-id=9 --scale-factor=1 --format=parquet
-  --query-id=9 --scale-factor=10 --format=native
-  --query-id=9 --scale-factor=10 --format=parquet
   --query-id=10 --scale-factor=1 --format=native
   --query-id=10 --scale-factor=1 --format=parquet
-  --query-id=10 --scale-factor=10 --format=native
-  --query-id=10 --scale-factor=10 --format=parquet
   --query-id=11 --scale-factor=1 --format=native
   --query-id=11 --scale-factor=1 --format=parquet
-  --query-id=11 --scale-factor=10 --format=native
-  --query-id=11 --scale-factor=10 --format=parquet
   --query-id=12 --scale-factor=1 --format=native
   --query-id=12 --scale-factor=1 --format=parquet
-  --query-id=12 --scale-factor=10 --format=native
-  --query-id=12 --scale-factor=10 --format=parquet
   --query-id=13 --scale-factor=1 --format=native
   --query-id=13 --scale-factor=1 --format=parquet
-  --query-id=13 --scale-factor=10 --format=native
-  --query-id=13 --scale-factor=10 --format=parquet
   --query-id=14 --scale-factor=1 --format=native
   --query-id=14 --scale-factor=1 --format=parquet
-  --query-id=14 --scale-factor=10 --format=native
-  --query-id=14 --scale-factor=10 --format=parquet
   --query-id=15 --scale-factor=1 --format=native
   --query-id=15 --scale-factor=1 --format=parquet
-  --query-id=15 --scale-factor=10 --format=native
-  --query-id=15 --scale-factor=10 --format=parquet
   --query-id=16 --scale-factor=1 --format=native
   --query-id=16 --scale-factor=1 --format=parquet
-  --query-id=16 --scale-factor=10 --format=native
-  --query-id=16 --scale-factor=10 --format=parquet
   --query-id=17 --scale-factor=1 --format=native
   --query-id=17 --scale-factor=1 --format=parquet
-  --query-id=17 --scale-factor=10 --format=native
-  --query-id=17 --scale-factor=10 --format=parquet
   --query-id=18 --scale-factor=1 --format=native
   --query-id=18 --scale-factor=1 --format=parquet
-  --query-id=18 --scale-factor=10 --format=native
-  --query-id=18 --scale-factor=10 --format=parquet
   --query-id=19 --scale-factor=1 --format=native
   --query-id=19 --scale-factor=1 --format=parquet
-  --query-id=19 --scale-factor=10 --format=native
-  --query-id=19 --scale-factor=10 --format=parquet
   --query-id=20 --scale-factor=1 --format=native
   --query-id=20 --scale-factor=1 --format=parquet
-  --query-id=20 --scale-factor=10 --format=native
-  --query-id=20 --scale-factor=10 --format=parquet
   --query-id=21 --scale-factor=1 --format=native
   --query-id=21 --scale-factor=1 --format=parquet
-  --query-id=21 --scale-factor=10 --format=native
-  --query-id=21 --scale-factor=10 --format=parquet
   --query-id=22 --scale-factor=1 --format=native
   --query-id=22 --scale-factor=1 --format=parquet
-  --query-id=22 --scale-factor=10 --format=native
-  --query-id=22 --scale-factor=10 --format=parquet
 
   To run all combinations:
   $ conbench tpch --all=true
 
 Options:
   --query-id [1|2|3|4|5|6|7|8|9|10|11|12|13|14|15|16|17|18|19|20|21|22]
-  --scale-factor [1|10]
+  --scale-factor [1]
```

Comment:
haha dang I dislike these tests

Reply:
💯 and not just the CLI help ones either, really; a lot of the benchmark tests are very inconsistent and don't give much guarantee they'll actually work. Whenever we eventually move off the legacy runner we should put some thought into how to test these better without a slow full-on benchmark run. Users are tests, but the feedback loop there is bigger than ideal.

Reply:
FWIW, the reason this test changes is because we're making this change in a slightly strange place (the validity setting function below). That's a fair enough place to do this so that we can get it done + shipped quickly (and since we do believe we will soon ™️ move over to a different runner anyway), but that's not really capturing the spirit of this, which is: the benchmark is _valid_ at scalefactors 1 and 10 (and 0.01! and 100!) it just will not complete successfully unless the machine has enough memory space.

It might be good to comment this test here as well that the behavior of it is _tied to the memory of the host running it_ now. That will (probably in most cases) be just fine, but if someone with >= 64gb of ram runs this test it will fail with this change.

Reply:
Yeah, this was bugging me, so I just imported `machine_info()` and skipped this test if it's on a machine with too much memory (with an explanation). It's still silly, but at least now it won't fail for silly reasons.